.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "prettier.configPath": "frontend/.prettierrc",
+    "prettier.ignorePath": "frontend/.prettierignore"
+}
```

Comment:
Sorry, real quick - was looking at some of the `prettier` additions.

Could having some of these configs in the `.vscode` configs cause some issues? Some individuals may not be using VSCode. Or if they are, could this conflict with any `settings.json` files that folks may already have? Just wondering

Reply:
Added an issue for it #363 and will work on this in the next days😃